hpolib/benchmarks/rl/learna.py

```diff
@@ -0,0 +1,362 @@
+        if configuration["conv_radius2"] == 0:
+            configuration["conv_size2"] = 0
+        del configuration["conv_radius2"]
+
+        if configuration["conv_size1"] != 0:
+            min_state_radius = configuration["conv_size1"] + configuration["conv_size1"] - 1
+            max_state_radius = 32
+            configuration["state_radius"] = int(min_state_radius
+                                         + (max_state_radius - min_state_radius) * configuration["state_radius_relative"])
+        else:
+            min_state_radius = configuration["conv_size2"] + configuration["conv_size2"] - 1
+            max_state_radius = 32
+            configuration["state_radius"] = int(min_state_radius
+                                         + (max_state_radius - min_state_radius) * configuration["state_radius_relative"])
+        del configuration["state_radius_relative"]
+
+        configuration["restart_timeout"] = None
+
+        return configuration
+
+
+class Learna(BaseLearna):
+
+    def __init__(self, data_path: Union[str, Path]):
+        super(Learna, self).__init__(data_path)
+    
+    @AbstractBenchmark._configuration_as_dict
+    def objective_function(self, configuration: Dict, cutoff_agent_per_sequence: Union[int, float, None] = 600, **kwargs) \
+            -> Dict:
+
+        config, network_config, agent_config, env_config = self._setup(configuration)
+
+        start_time = time()
+        validation_info = self._validate(evaluation_timeout=cutoff_agent_per_sequence,
+                                         restore_path=None,
+                                         stop_learning=False,
+                                         restart_timeout=config["restart_timeout"],
+                                         network_config=network_config,
+                                         agent_config=agent_config,
+                                         env_config=env_config)
+        cost = time() - start_time
+
+        return {'function_value': validation_info["sum_of_min_distances"],
+                'cost': cost,
+                'num_solved': validation_info["num_solved"],
+                'sum_of_first_distances': validation_info["sum_of_first_distances"],
+                }
+
+    @AbstractBenchmark._configuration_as_dict
+    def objective_function_test(self, configuration: Dict, cutoff_agent_per_sequence: Union[int, float, None] = 600,
+                                **kwargs) -> Dict:
+        return self.objective_function(configuration, cutoff_agent_per_sequence=cutoff_agent_per_sequence, **kwargs)
+
+
+class MetaLearna(BaseLearna):
+    def __init__(self, data_path: Union[str, Path]):
+        super(MetaLearna, self).__init__(data_path)
+        self.config = hpolib.config.config_file
+
+    @AbstractBenchmark._configuration_as_dict
+    def objective_function(self, configuration: Dict, cutoff_agent_per_sequence: Union[int, float, None] = 3600, **kwargs)\
```

Comment:
Shouldn't this also require a seed or is this method absolutely deterministic?

Reply:
Actually, the learna functions don't take seeds as input. And to be honest I don't see where they set the seed. 

I've tried to set the seeds manually in our code. But it doesn't seem to be possible to make tensorforce deterministic. 
That's somehow related to the cartpole benchmark. 

Maybe @Rungetf has an idea how to make learna deterministic.

Reply:
```
from tensorflow import set_random_seed
set_random_seed(0)
```
does not work.

Reply:
Hmm, but the Cartpole benchmark does at least accept a seed. Let me create an issue, so we can solve this in a different PR